src/Symfony/Component/Translation/Util/XliffUtils.php

```diff
@@ -129,7 +129,7 @@ public static function getErrorsAsString(array $xmlErrors): string
     private static function getSchema(string $xliffVersion): string
     {
         if ('1.2' === $xliffVersion) {
-            $schemaSource = file_get_contents(__DIR__.'/../Resources/schemas/xliff-core-1.2-strict.xsd');
+            $schemaSource = file_get_contents(__DIR__.'/../Resources/schemas/xliff-core-1.2-transitional.xsd');
```

Comment:
Is the introduction of `prop-group/prop` the hard requirement to change from strict to transitional?

https://www.diffchecker.com/XpX2SeQl

Reply:
Yep it is